src/main/java/core/basesyntax/db/FruitShopDao.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.db;
+
+import java.util.List;
+
+public interface FruitShopDao {
+    int storageSize();
+
+    List<String> getKeyAndValue();
```

Comment:
The method name 'getKeyAndValue' is ambiguous. Since this is a DAO for a fruit shop, a more descriptive name would be better. Consider renaming it to reflect that it returns fruit names and their quantities, such as 'getAllFruitsWithQuantities'.